core/drivers/crypto/se050/core/scp03.c

```diff
@@ -4,11 +4,11 @@
  * Author: Jorge Ramirez <jorge@foundries.io>
  */
 
-#include <crypto/crypto.h>
+#include <crypto/crypto_secure_element.h>
```

Comment:
For commit "core: crypto: Secure Element cryptographic interface": 
sorry for this late comment. what about _crypto_se.h_ for short? API function would shorten to `crypto_se_enable_scp03()`.

Reply:
it is just that I didnt want to use SE in case it generates confusion with Security Extensions...but I can change it if you think it will be ok

Reply:
We have used `_se` for Secure Element in the past, see commit e147a447.

Reply:
ok